<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parts Inventory Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../icons/icon.png</normaloff>../../../../../../icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1241</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Inventory Details</string>
     </attribute>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>1211</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MinArea</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MaxArea</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NbrOfHoles</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MinDiameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MaxDiameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>581</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search of reference with count level lower or equal to:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="count_filter_txt">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Inventory Statistics</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of parts type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Number of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Number of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ref_nbr">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_parts_nbr">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_hole">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>160</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_min_hole">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>40</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_min_hole_1">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>40</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>REI</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_max_hole_2">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>160</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Century Gothic&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>REI</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>1001</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>Reference</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Part Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>691</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Top 3 Reference with minimum inventory level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="check_btn">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>270</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Edit Inventory</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Part Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimum Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum Area</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>150</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Number Of Holes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>210</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimum Diameter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>270</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum Diameter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>330</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reference">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="part_name">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="min_area">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="max_area">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="max_diameter">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>280</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="min_diameter">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="number_of_holes">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>160</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Century Gothic&quot;;
background-color: rgb(171, 171, 171);
color: rgb(173, 66, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="count">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>340</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="first_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>410</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="last_btn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>410</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(243, 150, 255);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>410</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>410</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>410</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Century Gothic&quot;;
color: rgb(106, 253, 255);
background-color: rgb(147, 147, 147);</string>
    </property>
    <property name="text">
     <string>Parts Inventory Manager</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>610</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
